src/components/Modal/dialog.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { MODAL } from '../ThemeIdentifiers';
+import Helpers from './helpers';
+import * as baseTheme from './Modal.scss';
+
+export enum SizeType {
+  Small = 300,
+  Medium = 600,
+  Large = 900,
+}
+
+export type Size = 'Small' | 'Medium' | 'Large' | number;
+
+
+export interface SizeStyle {
+  width: string,
+  marginLeft: string,
+  left: string,
+}
+
+export interface Props {
+  close?: boolean,
+  children?: React.ReactNode,
+  footer?: React.ReactNode,
+  header?: string | React.ReactNode,
+  id?: string,
+  backdropEnabled?: boolean,
+  size?: Size,
+  theme?: any,
+  onClose?(e: any): void,
```

Comment:
We need to define the `e` parameter type for this, and all the functions below.  Use https://stackoverflow.com/a/42085792/487940.  @mkulinski Can you research a bit on what we should do?